django/http/response.py

```diff
@@ -50,6 +50,9 @@ def __init__(self, content_type=None, status=None, reason=None, charset=None):
         self.cookies = SimpleCookie()
         self.closed = False
         if status is not None:
+            if not isinstance(status, int) or status < 100 or status > 599:
+                raise ValueError('invalid HTTP status code: "{status}"'.format(
```

Comment:
It might be better to omit the quotes around status since that's making everything seem like a string.